resolwe/flow/executors/docker/seccomp.py

```diff
@@ -0,0 +1,1681 @@
+"""SECCOMP policy."""
```

Comment:
Could you expand this docstring?

It would be nice to mention how did you obtain this policy and an overview of what it blocks/allows.